fix(theme): updated theme and v4 to be consistent with PF styles

mcoker opened the pull request:
fixes https://github.com/patternfly/patternfly-org/issues/2887

@evwilkin in order for this to work properly, it's going to need to pull in @patternfly/patternfly@4.196.1 or greater, which adds a global variable for the light grey background the page sections on org use. Not sure if this would be part of the release process when we release the new versions or if we can get ahead of that by bumping versions in this PR to test? I tested everything locally so I was able to verify with the latest core.

patternfly-build commented:
PF4 preview: https://patternfly-org-pr-2929-v4.surge.sh/v4

nicolethoen commented:
Does this address this issue as far as you know? 
https://github.com/patternfly/patternfly-org/issues/2887

mcoker commented:
@nicolethoen it addresses a lot of #2887 though I didn't comb through everything, just took a quick pass through the CSS I could find in the repo. There may be more things to update that I didn't catch.

nicolethoen commented:
okay - it might be enough to call this issue closed once this PR is done/merged.
The next steps, as far as I'm aware, would be to add a theme switcher to the core/react workspaces, and in doing that we will likely find more issues we can follow up with in org.
@evwilkin does that sound good to you?

evwilkin commented:
@nicolethoen @mcoker thanks - yep the intention of that issue was to enable dark theme support, so if that's mostly the case and there are other random variables we uncover over time I think that's sufficient to close out the related issue.

Agreed that next steps would be adding a switcher to the workspaces so we have a resource to direct consumers to until we're ready to add dark theme to the website.  @mcarrano is there a designer who would want to review the background color changes on Org that @mcoker made?  I know we've got a few different grays, not sure how intentional these colors are.

mcarrano commented:
@mcoker @evwilkin I guess that I'm unclear what the goal is for changing the background color on the website. Is there a reason we need to do this? As far as design input, I'd like @mceledonia and @maryshak1996 to take a look at this.

mcarrano commented:
Ah, OK I get it now. Tomorrow is my weekly meeting with PF designers, so I put this on the agenda to discuss. @mcoker if you would like to be included, I'm happy to send an invite.

mcoker commented:
@mcarrano that would be great, thanks!

mcoker commented:
@mcarrano it does, but we need to pull in the latest version of core to see it. @evwilkin @nicolethoen are we able to do that in this PR?

mcoker commented:
@mcarrano it does, but we need to pull in the latest version of core to see it. @evwilkin @nicolethoen are we able to do that in this PR?

nicolethoen commented:
@mcoker if you rebase, you should see the latest version of core, now? if not, you are able to do this as part of this PR, for sure!

mcoker commented:
@nicolethoen do you know how I bump the version of core used on org? I looked through all of the `package.json` files in this repo and couldn't find `@patternfly/patternfly`

```
marvin:patternfly-org cmichael$ for i in `find . -name package.json | grep -vi node_modules`; do echo "looking in $i"; grep '@patternfly' $i; echo; done
looking in ./package.json

looking in ./packages/v4/package.json
    "@patternfly/react-docs": "5.82.5",

looking in ./packages/v3/package.json

looking in ./packages/ast-helpers/package.json
  "name": "@patternfly/ast-helpers",

looking in ./packages/theme-patternfly-org/package.json
    "@patternfly/ast-helpers": "^0.4.62",
```

evwilkin commented:
@mcoker 
```
git fetch upstream
git merge upstream/main
```

It's pulled in from the latest version of `react-docs` here:
- https://github.com/patternfly/patternfly-org/blob/main/packages/v4/package.json#L20

Also - I'm not sure that the new background color is being applied correctly, it looks like the rule that was updated removed the `#main-content` selector (which is still present) and kept the `.ws-docs-title` (which doesn't appear to be present):
- https://github.com/patternfly/patternfly-org/pull/2929/files#diff-fa41f8185de46180e16035ae00bea777bba247ed3590eb5af76be6c36ae9a77cL7-L9
- 
![Screen Shot 2022-06-15 at 4 00 07 PM](https://user-images.githubusercontent.com/8651509/173916913-b481a16f-e532-4ec1-ac28-0a432b6f23e4.png)

evwilkin reviewed (commented):
Rebased & reviewed locally, everything looks fantastic pending confirmation of the new background color variable being implemented correctly in the right places.

mcoker commented:
@evwilkin thanks! I didn't yarn install - ran `yarn clean && git clean -dfx` and rebuilding everything and looks like it pulled it in. Also updated that ws-docs-title page section and removed the class CSS

evwilkin reviewed (approved):
LGTM - thanks! 🏆

mcarrano reviewed (approved):
Looks good to me!
